lib/rules/no-restricted-imports.js

```diff
@@ -63,6 +63,9 @@ const arrayOfStringsOrObjectPatterns = {
                     message: {
                         type: "string",
                         minLength: 1
+                    },
+                    caseSensitive: {
+                        type: "boolean"
```

Comment:
Can you set the default here for clarity?

Reply:
I'm having trouble doing this.  When I tried:

```suggestion
                        type: "boolean",
                        default: false
```

I get a lot of errors like this during testing:

`Error: Schema for rule no-restricted-imports is invalid: default is ignored for: data3.caseSensitive`

Reply:
Ajv ignores defaults inside `anyOf` (https://github.com/eslint/eslint/pull/11427), which is the case here, thus this default wouldn't have any effects. RuleTester intentionally disallows ignored defaults (https://github.com/eslint/eslint/pull/11599) because relying on them can cause bugs, so we can't set default here.

Reply:
Ah, that explains it!

@nzakas: is there anything else you would want to do instead to clarify?

Reply:
Looks good.